test/unit/user/permissions_test.rb

```diff
@@ -150,4 +150,28 @@ class User::PermissionsTest < ActiveSupport::TestCase
     assert user.forbidden_some_services?
   end
 
+  SERVICE_ADMIN_SECTIONS = %i[partners plans monitoring]
+  NO_SERVICE_ADMIN_SECTIONS = %i[services portal finance settings]
```

Comment:
This should be dynamic from all the sections (declared in `app/models/admin_section.rb`) - the previously declared constant